src/Proxy/StoresPack.cs

```diff
@@ -68,6 +76,79 @@ public ProcessResult UnAllocateRequisition(int reqNumber, int? reqLineNumber, in
             };
         }
 
+        public void DoTpk(int locationId, int palletNumber, DateTime dateTimeStarted, out bool success)
+        {
+            success = false;
+            using (var connection = this.databaseService.GetConnection())
+            {
+                connection.Open();
+                var cmd = new OracleCommand("stores_oo.do_tpk", connection) //  TODO - Move function to stores_pack
```

Comment:
todo is done?

Reply:
I meant to ask - were you saying we should copy the actual PLSQL procedures from stores_oo into stores_pack? Or just keep them all in the same C# StoresPack proxy?

Reply:
stores_pack is just a wrapper for stores_oo. The stores_oo functions should just be called from in there.
It allows us to grab the messages generated in the session and translate booleans to int results etc